eduNEXT/tutor-contrib-edunext-distro
# Ds 172 add an option to disable current packages

This PR is to add an option to disable current or default packages

## How to test it

Optional: Prune system from docker with `docker kill $(docker ps -q)` and `docker system prune --volumes` 
1. Create tvm environment and install distro plugin on this branch.
  ```bash
  tvm project init mango v13.3.1
  source .tvm/bin/activate
  tvm plugins install git+https://github.com/eduNEXT/tutor-contrib-edunext-distro@DS-172-add-an-option-to-disable-current-packages
  ```
2. Now edit your config.yml adding some package with None value.

> example: `DISTRO_EOX_HOOKS_DPKG: None`

3. Save and start tutor.
```bash
tutor config save
tutor dev start
```

4. When is running you can check is some package is running  with `http://local.overhang.io:8000/PACKAGE-NAME/eox-info`. 
> ` http://local.overhang.io:8000/eox-hooks/eox-info` should fails if you disabled it.
